docker/mint/docker-run.sh

```diff
@@ -6,10 +6,12 @@ set -ex
 export RUST_BACKTRACE=full
 
 declare -a params
+if [ -n "${CFG_CHAIN_ID}" ]; then
```

Comment:
That's it, the final piece!  It explained why we saw different waypoints last night.